JASP-Engine/JASP/R/descriptives.R

```diff
@@ -16,6 +16,7 @@
 #
 
 Descriptives <- function(jaspResults, dataset, options) {
+.crashPlease()
```

Comment:
Travis fails because you left this in... :P

Reply:
Pretty good reason :p